spec/lib/mail_handler/mail_handler_spec.rb

```diff
@@ -336,6 +336,26 @@ def expect_header_string(fixture_file, header, header_string)
 
 describe 'when getting attachment attributes' do
 
+  it 'should handle an Outlook attachment with HTML generated from RTF' do
+    mail = get_fixture_mail('outlook-encoding-rtf.email')
+    attribute_hashes = MailHandler.get_attachment_attributes(mail)
+    attribute_hashes.each do |attribute_hash|
+      if attribute_hash[:body].respond_to?(:valid_encoding)
+        attribute_hash[:body].valid_encoding?.should be_true
+      end
+    end
+  end
+
+  it 'should handle an Outlook attachment with multiple attachments' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.